### PR TITLE
Add variant effect to PFS earplugs and correct links between effects and items

### LINKED ISSUE #4022: Effect: Earplugs does not work and links to wrong item
1. This item's rule elements do not seem to affect saving throws against audible things.
2. The description links to "Earplugs" rather than to "Earplugs (PFS Guide)" (which does include a link to "Effect: Earplugs").  The rule elements work with the PFS Guide rules, which are different than the usual Earplugs rules.

I think the solution should be to just delete the Effect: Earplugs and move the rules to the PFS guide item, to mirror the official item.  Alternatively, rename this to "Effect: Earplugs (PFS Guide)" and fix it.

### PR DESCRIPTION
**Pull request by TMunsie**
Closes #4022

### CODE REVIEW

File: packs/data/equipment-effects.db/effect-earplugs-pfs-guide.json [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+                "predicate": [
+                    "auditory"
+                ],
+                "selector": "saving-throw",
+                "type": "item",
+                "value": 1
+            },
+            {
+                "key": "FlatModifier",
+                "label": "Earplugs (hearing)",
```
Comment (TikaelSol) [inline packs/data/equipment-effects.db/effect-earplugs-pfs-guide.json:32]:
```suggestion
```
No labels needed

File: packs/data/equipment-effects.db/effect-earplugs-pfs-guide.json [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+            "unit": "unlimited",
+            "value": -1
+        },
+        "level": {
+            "value": 0
+        },
+        "rules": [
+            {
+                "key": "FlatModifier",
+                "label": "Earplugs (vs. auditory)",
```
Comment (TikaelSol) [inline packs/data/equipment-effects.db/effect-earplugs-pfs-guide.json:22]:
```suggestion
```

File: packs/data/equipment-effects.db/effect-earplugs-pfs-guide.json [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+                ],
+                "selector": "saving-throw",
+                "type": "item",
+                "value": 1
+            },
+            {
+                "key": "FlatModifier",
+                "label": "Earplugs (hearing)",
+                "predicate": [
+                    "hearing"
```
Comment (TikaelSol) [inline packs/data/equipment-effects.db/effect-earplugs-pfs-guide.json:34]:
```suggestion
                    "auditory"
```
Should this be `auditory` as well? If you were making an inline perception check you would likely use the standard trait name.

File: packs/data/equipment-effects.db/effect-earplugs-pfs-guide.json [hunk 4]
```diff
@@ -0,0 +1,59 @@
+{
+    "_id": "NE7Fm5YnUhD4ySX3",
+    "img": "systems/pf2e/icons/equipment/adventuring-gear/earplugs.webp",
+    "name": "Effect: Earplugs (PFS Guide)",
+    "system": {
+        "badge": null,
+        "description": {
+            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.iYiA6CMR1N2E2Ny2]{Earplugs (PFS Guide)}</p>"
```
Comment (TikaelSol) [inline packs/data/equipment-effects.db/effect-earplugs-pfs-guide.json:8]:
```suggestion
            "value": "<p>Grants a +1 item bonus to saves vs. auditory effects and a -2 penalty to perception checks that rely on hearing.</p><p>Granted by @UUID[Compendium.pf2e.equipment-srd.iYiA6CMR1N2E2Ny2]{Earplugs (PFS Guide)}</p>"
```

**Review by TMunsie — COMMENTED** [review]
Apply Tikael's additional suggestions

**Pull request merged**